README.md

```diff
@@ -138,7 +138,7 @@ In case you are using an older version of Lock for **Social** Authentication, th
 Make sure the Activity's `launchMode` is declared as `singleTask` or the result won't come back in the authentication.
 
 
-Then in any of your Activities you need to initialize **Lock**:
+Then in any of your Activities you need to initialize **Lock**. In case that your `auth0Scheme` value is different from `https` you should tell the library which scheme to use by calling `withScheme()` as pointed below.
```

Comment:
`If your `auth0Scheme` value is not `https` then you need to use `withScheme()` as shown below.